src/routes/openData.js

```diff
@@ -0,0 +1,40 @@
+const router = require('express').Router();
+const axios = require('axios');
+
+/**
+ * Get all events of a certain student
+ */
+router.route('/schedule/:courseId/:subject/:catalog').get(async (req, res) => {
+    res.set('Access-Control-Allow-Origin', '*');
+    try {
+        const scheduleData = await fetchConcordiaSchedule(req.params.courseId, req.params.subject, req.params.catalog)
+            console.log(`scheduleData`, scheduleData)
+            return scheduleData
+    } catch (err) {
+        res.status(400).json('Error: ' + err)
+    }
+    ;
+});
+
+/**
+ * Service to fetch current Concordia schedule for the given parameters
+ * @param courseId ID of unique course (ex. Course 002625 of BIOL201)
+ * @param subject Subject of course (ex. BIOL)
+ * @param catalog Course number (ex. 201)
+ * @returns {Promise<void>}
+ */
+const fetchConcordiaSchedule = async (courseId, subject, catalog) => {
+    await axios.get(`https://opendata.concordia.ca/API/v1/course/schedule/filter/${courseId}/${subject}/${catalog}`, {
+        auth: {
```

Comment:
API credentials should not be left in the code. Please move to local .env file.

Reply:
No problem man, let me know when you are ready to merge and we can pair to implement a final structure for Concordia Open Data :)

Reply:
I think I am going to close this PR? I have just reviewed your branch data_fetch_ConU_open_data, and it seems you have already done the route for academic events and openData connection in the best way possible!